cmake/Modules/FindPoco.cmake

```diff
@@ -195,6 +195,10 @@ if(Poco_INCLUDE_DIR)
         if(LIB${COMPONENT})
           list(APPEND Poco_LIBRARIES "${LIB${COMPONENT}}")
           list(APPEND Comp_List "${COMPONENT}${DBG}")
+        else()
+          message(STATUS "  Poco found, but missing component 'Poco${COMPONENT}${DBG}'...")
+          message(STATUS "  This can happen when poco is found in release, but debug is required.")
```

Comment:
The message describe one case (found release, requiring debug) but can also be triggered for the opposite case. Either the message should cover both cases or a different message should be shown based on `DBG`.

Reply:
The newly added second message is the same as the first one?

Reply:
Doh, fixed it.